components/gitpod-db/src/typeorm/user-db-impl.ts

```diff
@@ -416,7 +416,7 @@ export class TypeORMUserDBImpl extends TransactionalDBImpl<UserDB> implements Us
             const envVarRepo = ctx.entityManager.getRepository<DBUserEnvVar>(DBUserEnvVar);
 
             await envVarRepo.update(
-                { id: envVar.id, userId },
+                { id: envVar.id, userId, deleted: false },
```

Comment:
@mustard-mh is it failing before or after this change?

Reply:
Error is throw by code above `An environment variable with this ID could not be found`. When I update user env value only

Reply:
We don't pass `envId` here

Reply:
Same on `envvar-service-api` side

Reply:
👍

Reply:
Fixed